src/content/en/tools/lighthouse/index.md

```diff
@@ -70,103 +76,138 @@ failing tests as indicators on what you can do to improve your app.
   </a>
 </p>
 
-Note: Lighthouse currently has a large focus on Progressive Web App features,
-such as [Add to Homescreen](/web/fundamentals/engage-and-retain/app-install-banners/)
-and offline support. However, the overarching goal of the project is to provide
-an end-to-end audit of all aspects of web app quality.
+## Get started {: #get-started }
 
-## Setting up Lighthouse
+There are two primary ways to run Lighthouse: as a Chrome Extension, or as a
+command line tool. The Chrome Extension provides a more user-friendly
+interface for reading reports. The command line tool enables you to integrate
+Lighthouse into continuous integration systems.
 
-There are two primary ways to run Lighthouse, as a Chrome Extension, or as a command
-line tool. The Chrome Extension provides a more user-friendly interface for
-reading reports. The command line tool enables you to integrate Lighthouse into
-continuous integration systems.
+### Install and run the Chrome Extension {: #extension }
 
-### Install the Chrome Extension
+To install the extension:
 
-To install the extension, do the following:
+1. Download [Google Chrome for Desktop][desktop].
 
-1. Download Google Chrome 54 or later.
-- Install the <a class="gc-analytics-event"
-     data-category="crx-install" data-label="lighthouse-install-button"
-     href="https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk"
-     title="Install Lighthouse Chrome Extension" target="_blank">
-    <span class="material-icons" style="vertical-align:middle">file_download</span>
-    Lighthouse Chrome Extension</a> from the Chrome Webstore.
+1. Install the <a class="gc-analytics-event" data-category="Lighthouse"
+   data-label="Home / Install Extension" href="https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk"
```

Comment:
do you want to update the category/labels on the main install button too (one at the top)?